examples/tutorials/vectors.py

```diff
@@ -0,0 +1,459 @@
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
+"""
+# sphinx_gallery_thumbnail_number = 6
+
+import numpy as np
+import pygmt
+
+########################################################################################
+# Plot Caretesian Vectors
+# -----------------------
+#
+# Create a simple Cartesian vector using a starting point through
+# ``x``, ``y``, and ``direction`` parameters. The direction is specified
+# by a list of two 1d arrays structured as ``[[angle_in_degrees], [length]]``
+#
+# On the shown figure, the plot is projected on a 10cm X 10cm region,
+# which is specified by the ``projection`` parameter.
+# The length of the vector also uses centimeters by default but
+# could be changed using :meth:`pygmt.config`
+# (Check the next examples for unit changes).
+#
+# Notice that the ``v`` in the ``style`` parameter stands for
+# vector; it distinguishes it from regular lines and allows for
+# different customization. ``0c`` is used to specify the size
+# of the arrowhead which explains why there is no arrow on either
+# side of the vector.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    x=2,
+    y=8,
+    direction=[[-45], [6]],
+    style="v0c",
+)
+fig.show()
+
+########################################################################################
+# In this example, we apply the same concept shown previously to plot multiple
+# vectors. Notice that instead of passing int/float to ``x`` and ``y``, a list
+# of all x and y coordinates will be passed. Similarly, the length of direction
+# list will increase accordingly.
+#
+# Additionally, we changed the style of the vector to include a red
+# arrowhead and increased the thickness of the line. A list of different
+# styling attributes can be found in
+# :doc:`Vector heads and tails </gallery/lines/vector_heads_tails>`.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    x=[2, 4],
+    y=[8, 1],
+    direction=[[-45, 23], [6, 3]],
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# The default unit of vector length is centimeters,
+# however, this can be changed to inches or points. Note that, in PyGMT,
+# one point is defined as 1/72 inch.
+#
+# In this example, the graphed region is 10in X 4in, but
+# the length of the first vector is still graphed in centimeters.
+# Using ``pygmt.config(PROJ_LENGTH_UNIT="i")``, the default unit
+# can be changed to inches in the second plotted vector.
+
+fig = pygmt.Figure()
+# Vector 1 with default unit as cm
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10i/4i",
```

Comment:
I would prefer to have a smaller figure (smaller image size and faster execution time), and keep the same scale for X and Y axes.
```suggestion
    projection="X5i/5i",
```